### DISCUSSION TITLE
NeuroSpell's chunker using spaCy

### DISCUSSION THREAD

**Discussion by EtienneAb3d** [explosion/spaCy#3675] [Show & Tell]
Hi all,

here is just a short message to let you know that:
- few weeks ago, I started writing NeuroSpell, an automatic corrector using a neural network. It uses spaCy (at this moment, only for NER). Freely available for online use:
http://neurospell.com/en.php
- today, I started releasing some parts of its code as open-source. The first released part is a chunker using the spaCy server:
https://github.com/EtienneAb3d/OpenNeuroSpell

Best regards,
Etienne

**Comment by BramVanroy** [comment]
Even though this is a fun project, Github issues should not be used for this kind of updates/promotions. Instead you may want to add it to the [spaCy universe](https://github.com/explosion/spaCy/blob/master/website/meta/universe.json). But considering that your implementation just uses spaCy's server, and does everything else from a Java rule-based approach, I'm not sure how well-suited is for the Universe. Maybe @ines can weigh in on this.

**Comment by svlandeg** [comment]
I realise this is a very old issue :-) But we've just now created a "Show and Tell" subforum on the new GH Discussions page, so that we'd have a proper place to share and discuss spaCy-related projects and plugins!
